configs/base.yaml

```diff
@@ -2,34 +2,57 @@ output_dir: output
 
 samples: &samples
   GN2ej:
-    path: '/Users/jburzyns/Documents/work/ATLAS/EmergingJets/test_samples/epoch=015-val_loss=0.02852__test_pp_output_val.h5'
+    path: '/home/randon_h/gnnej/gnn-plots/data/epoch=008-val_loss=0.02631__test_pp_output_val.h5'
     label: 'GN2ej'
     reference: True
     df_name: jets
   GN2ej_noPt:
-    path: '/Users/jburzyns/Documents/work/ATLAS/EmergingJets/test_samples/epoch=015-val_loss=0.02852__test_pp_output_val.h5'
+    path: '/home/randon_h/gnnej/gnn-plots/data/epoch=008-val_loss=0.02631__test_pp_output_val.h5'
     label: 'GN2ej_noPt'
     reference: False
     df_name: jets
 
 style: &style
   ylabel: ""
   xlabel: ""
-  figsize: !!python/tuple [6.5, 6]
+  figsize: !!python/tuple [6, 6]
+  use_atlas_tag: False
```

Comment:
This config was meant to just give an example of how to run things -- it would be better to not update this file directly and instead to keep a local version of the config that you are using outside of the repo